app/views/CannotSubmitLockedEventsView.scala.html

```diff
@@ -28,7 +28,7 @@ <h1 class="govuk-heading-xl">@messages("cannotSubmitLockedEvents.multiple.headin
         <p class="govuk-body">@messages("cannotSubmitLockedEvents.multiple.p1")</p>
         <ul class="govuk-list govuk-list--bullet">
             @seqOfEventTypes.map { eventSummary =>
-                <li>@messages("cannotSubmitLockedEvents.multiple.item", eventSummary.lockedBy.map(lockedBy => lockedBy).getOrElse(""), eventSummary.eventType.toString, eventSummary.recordVersion)</li>
+                <li>@messages("cannotSubmitLockedEvents.multiple.item", eventSummary.lockedBy.map(lockedBy => lockedBy).getOrElse(""), eventSummary.eventType.toString)</li>
```

Comment:
For the case where the name of the user locking the event can't be found, we'd just be displaying `must close event X`. Should we consider adding a default string to the `getOrElse` here like "Another user"?